sdk/internal/recording/recording.go

```diff
@@ -778,6 +758,12 @@ func Stop(t *testing.T, options *RecordingOptions) error {
 			// test should only be run live, don't want to generate recording
 			return nil
 		}
+		if testStruct.recordingSeed != 0 {
```

Comment:
How did this used to work, or was it broken?

Reply:
For the test-proxy testing framework, we don't have utility to support generating random string. So, we need to store the random seed in the option's variable and stop with this option to store the random seed. For now, we add such feature, so we need to do the seed store for user.